FreeRTOS/FreeRTOS
# Update M33F simulator Keil example

Description
-----------
 The example is updated to use latest CMSIS 5.9.0 and Device startup 2.1.0.

Test Steps
-----------

Follow the instructions in ReadMe.txt to run/debug the example.

Checklist:
----------


- [ x ] I have tested my changes. No regression in existing tests.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
